src/sidebar/store/modules/activity.js

```diff
@@ -64,16 +99,26 @@ function isLoading(state) {
   return state.activity.activeApiRequests > 0;
 }
 
+/**
+ * Return true when annotations are actively loading.
```

Comment:
I think "loading" is ambiguous in this context because it could refer to just fetching annotations from the server, but it could also refer to other work that has to happen once annotations are fetched - sending them to the host page, waiting for them to anchor, and waiting for the results to be received.

Reply:
It's issuing an annotation fetch api request, loading the results into the store, and marking the fetch status of the frame as complete. I guess `fetch` would be most appropriate here.